src/components/GameTile.vue

```diff
@@ -0,0 +1,354 @@
+<script setup>
+import { reactive, defineProps, computed } from "vue";
+import CustomButton from "@/components/CustomButton.vue";
+
+const props = defineProps({
+  tile: {
+    type: Object,
+    required: true,
```

Comment:
Adding default value for showing what object you expect here would be nice :)

```suggestion
    required: true, 
    default: () => ({
      id: 1,
      genres: [
        { name: "Adventure" },
        { name: "RPG" }
      ],
      platform: "apple",
      imagePath: "Cyberpunk2077.jpg",
      title: "CyberPunk 2077",
      price: 159.99,
      discountInPercentage: 0,
      reviews: 2137,
      star: 3,
      description: `Lorem ipsum dolor sit`,
    }),
```